packages/frontend/src/components/dynamic/album/AlbumDetails.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { type Song } from "@prisma/client";
+import Image from "next/image";
+import Link from "next/link";
+
+interface AlbumDetailsProps {
+  albumCoverImage: Song[] | undefined;
```

Comment:
if this is an array value best with a plural name.

Reply:
I'll sync the new changes on next pr